libs/theme/src/abstract.ts

```diff
@@ -86,7 +88,17 @@ export interface CustomTheme extends CommonTheme {
   welcome?: {
     background?: BackgroundStyle;
     image?: string;
+    imagePosition?: ImagePosition;
+    buttons: WelcomeButtons[];
```

Comment:
Now it probably makes sense to make `welcome` required if you don't provide fallback `buttons`.